services/user.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+
+import settings
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]